packages/website/src/components/SelectionGridItem.tsx

```diff
@@ -17,11 +17,15 @@ const SelectionGridItem = ({
     key={link}
   >
     <a className="SelectionGrid__link" href={link}>
-      <OneImageBackground className="SelectionGrid__image" src={imageURL}>
-        <div className="SelectionGrid__image-inside">
-          <div className="SelectionGrid__title">{title}</div>
-        </div>
-      </OneImageBackground>
+      <OneImage
+        className="SelectionGrid__image"
+        src={imageURL}
+        alt=""
+        aspectRatio={AspectRatio.r20by9}
+      />
+      <div className="SelectionGrid__image-inside">
```

Comment:
image-inside is used when we're wanting to place an image on top of an background via `position: absolute`, we don't need this if it's just beneath